packages/utils/generics/copy-to-clipboard.scss

```diff
@@ -0,0 +1,32 @@
+pre {
+  background-color: var(--pf-c-code-block--BackgroundColor);
+  color: var(--pf-global--Color--100);
+  display: block;
+  font-family: var(--pf-c-code-block__pre--FontFamily);
+  font-size: var(--pf-c-code-block__pre--FontSize);
+  overflow-wrap: break-word;
+  padding: var(--pf-global--spacer--md);
+  white-space: pre-wrap;
+}
+
+.co-copy-to-clipboard__text {
+  padding-right: 50px;
+}
+
+.co-copy-to-clipboard__btn,
+.co-copy-to-clipboard__btn:hover {
+  --pf-c-button--PaddingTop: 11px;
+  --pf-c-button--PaddingRight: 16px;
+  --pf-c-button--PaddingBottom: 11px;
+  --pf-c-button--PaddingLeft: 16px;
+  --pf-c-button--BorderColor: #ccc;
+  --pf-c-button--BorderRadius: 0;
+  --pf-c-button--hover--BorderWidth: 1px;
+  --pf-c-button--LineHeight: 20px;
+}
+
+.pf-c-clipboard-copy__group-copy {
+  position: absolute;
+  right: 0;
+  top: 3px;
+}
```

Comment:
do we have to copy pf classes as well ? did u try without coping this class once, if it is working as expected ?

Reply:
cause we use pf as a dependency, it should be fine IMO... if it is not a pf class then let's change it's name to just `clipboard-copy__group-copy`

Reply:
yeah I have tried, without it, it looks like below 

![image](https://user-images.githubusercontent.com/22158580/187402869-7c1a1bea-7cab-4c16-849f-0dcb45b8c935.png)

Reply:
okay then change the name of class as mentioned above...

Reply:
Done